index.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset='utf-8'>
+  <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+  <title>Portfolio</title>
+  <meta name='viewport' content='width=device-width, initial-scale=1'>
+  <link rel='stylesheet' type='text/css' media='screen' href='style.css'>
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0" />
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@400;500;600;700&family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
+<body>
+  <header>
+    <nav class="toolbar" id="toolbar">
+      <a hre="#" class="logo">My Logo</a>
+      <div class="menu">
+        <button class="dropdown-button"><span class="material-symbols-outlined">menu</span>
+        </button>
+        <ul class="dropdown-menu">
+          <a href="#work-section">Portfolio</a>
+          <a href="#about-section">About</a>
+          <a href="#">Contact</a>
+          <a href="#"><span class="material-symbols-outlined">mail</span></a>
+        </ul>
+      </div>
+    </nav>
+  </header>
+  <section class="headline">    
+    <h1>Hey There.<br>I'm Tomás</h1>
```

Comment:
- [ ]  Here is an example of an inline comment. Please make changes @tomasesquivelgc